learn/educators/python.md

```diff
@@ -0,0 +1,98 @@
+
+Professor Dodds has been recommending and using Visual Studio Code in his classes since Visual Studio Code debuted. "[Visual Studio Code] is the best balance of authenticity and accessibility"; it is a free professional-grade editor used by professional software engineers and students alike. "Plus, [Visual Studio Code] works on every OS": Windows, MacOS, Linux, and even [Chromebooks](https://code.visualstudio.com/blogs/2020/12/03/chromebook-get-started).
+
+Since Visual Studio Code is super customizable, Professor Dodds gets his students running a tailored experience for his class that hides suggestive code completions so that they learn from what they type and reinforce the conceptual models being built. Here are the settings his students use to achieve this but you can find the most up-to-date settings on his course website: [CS5 - Python Tips)](https://www.cs.hmc.edu/twiki/bin/view/CS5/PythonTips).
+
+```json
+"editor.quickSuggestions": false,
+"editor.acceptSuggestionOnCommitCharacter": false,
+"editor.suggest.filterGraceful": true,
+"editor.suggestOnTriggerCharacters": false,
+"editor.acceptSuggestionOnEnter": "on",
+"editor.suggest.showIcons": false,
+"editor.suggest.maxVisibleSuggestions": 7,
+"editor.hover.enabled": false,
+"editor.hover.sticky": false,
+"editor.suggest.snippetsPreventQuickSuggestions": false,
+"editor.parameterHints.enabled": false,
+"editor.wordBasedSuggestions": true,
+"editor.tabCompletion": "on",
+"extensions.ignoreRecommendations": true,
+"files.autoSave": "afterDelay",
+```
+
+
+Finally, he also utilizes the built-in terminal heavily in his class as an introduction to running programs from the command line and navigating around their machine. He appreciates how "the built-in terminal panel does not try to automate too much (which, if it did, would deprive newcomers of the experience of the information-flow that's going on)."
+
+Thank you, Professor Dodds, for sharing your story! If you’re interested in using Visual Studio Code to teach Python in your classes, you can get started with the Python Education Extension Pack above!
+
+## Coding Pack for Python (Windows)
+
+This coding pack includes everything you need to start coding in Python:
+- VS Code
+- Python and LiveShare extensions in VS Code
+- Python 3.8.5 and add it to the PATH
+- Python packages including, jupyter, numpy, sklearn, pandas, and matplotlib
+
+Full details and a link to install is available under [Coding Packs](/installers#_coding-pack-for-python).
+
+## Microsoft Learn free Python programming and data science lessons
+
+### NASA-Inspired Lessons
+This learning path enables students to use Python to explore doing analyses and projects inspired from real-world problems faced by National Aeronautics and Space Administration (NASA) scientists. View full details of the lessons under [NASA-Inspired Lessons](https://code.visualstudio.com/learn/students/nasa-python)
+
+![NASA Inspired Python lessons](images/python/nasa-learning-path.png)
```

Comment:
For the MS Learn images, the other /learn pages have used Dark theme, for example https://code.visualstudio.com/learn/educators/learn-modules.
Dark theme images show up better. If you do want to use Light theme, you should put a 3 pixel gray border around the image so it is outlined. The Windows Snipping Tool does this by default when you grab a screen shot